en/finding-sorting-and-cleaning-entries/groups.md

```diff
@@ -1,132 +1,268 @@
 
-**Using a free-form search expression**  
-This is similar to the above, but rather than search a single field for a single search term, the [search expression syntax](search.md#advanced) can be used, which supports logical operators \(`AND`, `OR`, `NOT`\) and allows to search multiple fields. For example, the search expression `keywords=regression and not keywords=linear` groups entries concerned with non-linear regression.
+#### Searching for a keyword in a field
 
-In the groups view, dynamic groups are shown in _italics_.
+This method groups entries in which a specified *field* (e.g. *author*) contains a specified *keyword* (e.g. *Smith*).
+The mentioned example will group all entries referring to the author *Smith*.
 
-#### Hierarchical context
+**%%% screenshot here: grab of the group type %%%**
 
-By default, a group is **independent** of its position in the groups tree: When selected, only the group's contents are shown. However, especially when using dynamic groups, it is often useful to define a subgroup that **refines its supergroup**, i.e., when selected, entries contained in both groups are displayed. For example, create a supergroup containing entries with the keyword `distribution` and a subgroup containing entries with the keyword `gauss` that refines this supergroup. Selecting the subgroup now displays entries that match both conditions, i.e. are concerned with Gaussian distributions. Note that items that only belong to the subgroup `gauss` will not be shown, i.e. for an item to be displayed when selecting `gauss` it is necessary to be assigned to both the subgroup `gauss` and the supergroup `distribution`. By adding another refining subgroup for `laplace` to the original supergroup, the grouping can easily be extended. In the groups tree, refining groups have a special icon \(this can be turned off in the preferences\).
+The search can be case-sensitive or not (checkbox 'Case sensitive').
+The search can either be done as a plain-text or a regular-expression search (checkbox 'Regular expression').
 
-The logical complement to a refining group is a group that **includes its subgroups**, i.e. when selected, not only the group's own entries, but also its subgroups' entries are shown. In the groups tree, this type of group has a special icon \(this can be turned off in the preferences\).
+Obviously, this will work only for entries including the specified grouping field, and the quality of the grouping will depend on the content accuracy.
 
-In other words, this defines which entries are displayed when this group is selected:
+**%%%Help needed: True in 5.2? I do not understand the paragraph... So, if true in 5.2 and unclear to you please reword %%%**
+In the former case, JabRef allows to manually assign/remove entries to/from the group by simply appending/removing the search term to/from the content of the grouping field.
```

Comment:
I think what this means is that JabRef will show the entry automatically in a group that matches entries with keyword `A` if you add `A` in the keyword field of that entry. This is not specific to this group: as soon as an entry is changed so that it now matches the criteria of a group, it's added to that group automatically.

Reply:
OK, I kept a modified version of it.